README.md

```diff
@@ -1,11 +1,112 @@
+
+	* Are the changes you want to make potentially applicable to a broad range of usage scenarios, including potential usage outside of qz.com?
+	* Is it the responsibility of this component to do the thing you want it to do?
+	* If you are adding some logic to this component, is this the right place in the tree to do so?
+	* If you are making changes based on a design, have you confirmed with the designer that these changes should be reflected in Prism UI?
+
+If you want to **add a new component**:
 
-<MyComponentName />
+	* Is this component functionally or visually distinct from other components in the library? Could the same effect be achieved by tweaking an existing component?
+	* Do you anticipate this component being long-lived and/or being used in more than one location?
+	* If you are adding a component from a design, have you confirmed with the designer that this component should be part of Prism UI?
+
+If the answer to any of these questions is no, chat with your teammates about how best to achieve what you want. Don’t be afraid to ask questions in the `#product-prism-support` Slack channel.
+
+### Running Storybook locally
+
+To run a local instance of Storybook, use `npm start`. Hot-reloading is supported.
+
+### Testing components in qz-react
+
+You may want to test your changes in a consuming repository, e.g. `qz-react`, before opening a pull request. The easiest way to do this is to publish a branch of this repository with your changes, e.g.
+
+* `git checkout -b my-new-component`
+* `git commit -am "My new component WIP"`
+* `git push -u origin/my-new-component`
+
+In your consuming repository, you can then install this branch as a dependency and import your component:
+
+* `npm i github:Quartz/interface#my-new-component`
+* `import { MyNewComponent } from '@quartz/interface/src/components' `
+
+### Updating qz-react with changes from Prism UI
+
+Once your pull request to Quartz/interface has been merged, take the commit hash containing your changes and in qz-react run:
+
+`npm i --save github:Quartz/interface#[YOUR_COMMIT_HASH]`
+
+Then open a second pull request in qz-react with the updated `package.json` and `package-lock.json` files.
+
+### Component checklist
+
+Before opening a pull request, please review the following checklist:
+
+#### Required
+
+* All code must pass linting and unit tests (`npm test`)
+* Components must be documented with a `[COMPONENT_NAME].story.jsx` and `[COMPONENT_NAME].docs.mdx` file
+* All tests must pass in the Storybook accessibility panel
+* propTypes must be documented using JSDoc format, e.g.
+```js
+/**
+	 * Whether the button is interactive. Forwarded to the button element.
+	 */
+	disabled: PropTypes.bool.isRequired,
 ```
+* If a prop’s sole purpose is to forward a value to an HTML attribute or React event handler, use the camel-cased name of that attribute, e.g. a prop that populates an `aria-label` attribute and nothing else should be named `ariaLabel`
```

Comment:
not sure i follow this one. what is the alternative?

Reply:
in this specific example, the prop was called `label`. We also have a few examples of props called things like `handleSubmit` being forwarded to `onSubmit`. I can try and make this a bit clearer